internal/compiler/generator/cpp.rs

```diff
@@ -951,16 +952,11 @@ fn generate_item_tree(
     target_struct.members.push((
         Access::Private,
         Declaration::Function(Function {
-            name: "parent_item".into(),
-            signature: "(slint::private_api::ComponentRef component, uintptr_t index, slint::private_api::ItemWeak *result) -> void".into(),
+            name: "parent_node".into(),
+            signature: "([[maybe_unused]] slint::private_api::ComponentRef component, [[maybe_unused]] slint::private_api::ItemWeak *result) -> void".into(),
             is_static: true,
             statements: Some(vec![
-                format!("auto self = reinterpret_cast<const {}*>(component.instance);", item_tree_class_name),
```

Comment:
Why remove this line?  
It made more sense to have it on a separate line rather than huge long lines.

Reply:
I wanted to merge it with the other line since on its own it produces warnings.